airbyte-integrations/bases/base-java/javabase.sh

```diff
@@ -16,7 +16,7 @@ if [[ $IS_CAPTURE_HEAP_DUMP_ON_ERROR = true ]]; then
   fi
 fi
 #30781 - Allocate 32KB for log4j appender buffer to ensure that each line is logged in a single println
-JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768"
+JAVA_OPTS=$JAVA_OPTS" -Dlog4j.encoder.byteBufferSize=32768 -Dlog4j2.configurationFile=log4j2.xml"
```

Comment:
Won't this screw up the logging during integration tests? We want those to be textual.